src/lib/Player.ts

```diff
@@ -127,6 +128,7 @@ export class Player extends EventEmitter {
      * @param position Seeks the song to the position specified in milliseconds, use the duration of the song from lavalink to get the duration
      */
     public seek(position: number): Promise<boolean> {
```

Comment:
I think, this is a better idea, so it doesn't emit seek if the send doesn't actually send to lavalink. just to be safe
```suggestion
    public async seek(position: number): Promise<boolean> {
        const d = await this.send("seek", { position });
        if (this.listenerCount("seek")) this.emit("seek", position);
        return d;
    }
```